gratipay/models/package/__init__.py

```diff
@@ -78,7 +95,9 @@ def ensure_team(self, cursor, owner):
                                )
             cursor.run('INSERT INTO teams_to_packages (team_id, package_id) '
                        'VALUES (%s, %s)', (team.id, self.id))
-        return team
+            break
+        else:
+            raise OutOfOptions()
```

Comment:
This code will never run, unless UUID4 is broken. I'd put my money on UUID4 and remove the else and the exception.